.version

```diff
@@ -1 +1 @@
-0.9.6
+0.9.7
```

Comment:
If this particular PR is going to be merged, then this version needs to be fixed as `0.9.7` has already been used. If #137 will be used instead, and this PR is abandoned, then it's already handled in that PR.